Makefile

```diff
@@ -229,8 +208,7 @@ check-licenses: ## Checks license status.
 check-prereqs: ## Checks prerequisite software status.
 	./scripts/prereq-check.sh
 
-# TODO: remove govet and gofmt checks once golangci-lint is being enforced.
-check-style: govet gofmt check-licenses check-plugin-golint ## Runs govet and gofmt against all packages and also ensures plugin package golint compliant
+check-style: golangci-lint plugin-checker check-licenses check-plugin-golint ## Runs golangci against all packages and also ensures plugin package golint compliant
 
```

Comment:
We can also remove the `megacheck` command. It is broken anyway.

Reply:
Yep, saw that. I will do it in the PR which deals with the tools dependency since megacheck is a tool.